integration/consul_catalog_test.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os/exec"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	checker "github.com/vdemeester/shakers"
+	check "gopkg.in/check.v1"
+)
+
+// Consul catalog test suites
+type ConsulCatalogSuite struct {
+	BaseSuite
+	client *api.Client
+}
+
+func (s *ConsulCatalogSuite) SetUpSuite(c *check.C) {
+	s.createComposeProject(c, "consul_catalog")
+	s.composeProject.Up()
+
+	config := api.DefaultConfig()
+	config.Address = "127.0.0.1:8500"
```

Comment:
Ok. sounds like a good plan. I guess that might be quite a bit in the future though so I'll try to get something like that but a bit more hardcoded for this case for now.